### DISCUSSION TITLE
Unable to deploy v.8

### DISCUSSION THREAD

**Discussion by zetle** [NHAS/wag#156] [General]
Hello, I'm tring to deploy WAG on clean ubuntu server. No matter what configuration I try, the final result is:
```
root@wireguard-wag:/opt/wag# ./wag start -config config.json
2024/12/12 09:07:13 no listen url was specified for the etcd cluster TLS manager, new wag servers will not be able to join.
```
I do not plan to use the clustering function at all.

**Comment by DyonB** [comment]
Hi, I've trying to get WAG up and running as well. Seems like a very cool project. Been using openvpn and never liked the lack of additional authentication for wireguard, thus this seems perfect.
But no luck so far. I'm unexperienced in this particular application with wireguard and tried to follow the instructions and other post, e.g. https://github.com/NHAS/wag/discussions/98 , best to get something working.
I now saw this post, and I get the same result.
I suppose there isn't a even elaborate more step by step how-to to for this project? 
Tomorrow I will try a older version, see if that works.

**Reply by DyonB** [reply]
hmm just tried v7.3.2 and got something going on the first try. Spend quite a few hours on the v8 trying to get it up, thinking I did something wrong. Maybe something in the config related to the cluster manger needed to be set correctly, but could not find that in the readme. 
Now off the tune the config file :)

**Reply by NHAS** [reply]
Hi there. 

That error occurs when  the TLSManagerListenURL is not specified in the clustering block. 

The name of the clustering block is a little misleading. As it's just what sets up an etcd member for wag to store it's data in. 

If you set TLSManagerListenURl to https://localhost:3333

Or something on localhost then that will mean clustering is effectively disabled as nothing will be able to join. 

I would suggest being on version 8 if possible. 
```json
   "Clustering": {
        "ClusterState": "new",
        "ETCDLogLevel": "error",
        "Witness": false,
        "TLSManagerListenURL": "https://wag.server:3434"
    },

```
